src/tests/fs_tests.rs

```diff
@@ -6,9 +6,11 @@ pub mod fs_tests {
     use crate::interface;
     use crate::safeposix::syscalls::fs_calls::*;
     use crate::safeposix::{cage::*, dispatcher::*, filesystem};
-    use libc::c_void;
+    use libc::{c_void, EINVAL};
+    use std::borrow::{Borrow, BorrowMut};
```

Comment:
Are we using these imports including the "Duration" and "Instant" ones in the tests, currently I don't see their references.

Reply:
Thanks for pointing that out!